src/code/z_kaleido_manager.c

```diff
@@ -39,7 +39,8 @@ void* KaleidoManager_FaultAddrConv(void* address, void* param) {
 
 void KaleidoManager_LoadOvl(KaleidoMgrOverlay* ovl) {
     ovl->loadedRamAddr = sKaleidoAreaPtr;
-    Load2_LoadOverlay(ovl->vromStart, ovl->vromEnd, ovl->vramStart, ovl->vramEnd, ovl->loadedRamAddr);
+    Load2_LoadOverlay(ovl->vromStart, ovl->vromEnd, (uintptr_t)ovl->vramStart, (uintptr_t)ovl->vramEnd,
```

Comment:
We usually try to use `void*` to hold actual vram pointers. I guess I could change the struct members to be `uintptr_t`

Reply:
Actually no, I think i prefer the members to be `void*`. Also I managed to get this function to match with the `void*` parameters instead of `uintptr_t`, so I'll keep this members as `void*`